lib/teaspoon/formatter/base.rb

```diff
@@ -154,6 +154,25 @@ def filename(file)
         filename += "?#{params.join("&")}" if params.any?
         filename
       end
+
+      def parse_output_file(output)
+        return output unless output
+        match = output.scan(/%{([^}]*)}/)
+        match.flatten.each do |m|
+          rep = case m
+                when "suite_name"
+                  @suite_name.underscore
+                when "date"
+                  Date.now.to_i
+                else
+                  warn "Teaspoon::Formatter - Output File can only contain the placeholders %{suite_name} or %{date}."
```

Comment:
I had to look up warn, cause I've never used it. Interesting, and probably a good use for it.